docs/styles/spec/file-input-spec.less

```diff
@@ -1,5 +1,5 @@
 @import (reference) 'components/button/index';
-@import (reference) 'components/file/index';
+//@import (reference) 'components/file/index';
```

Comment:
FYI: doing this breaks the File Input tests. If it's not corrected in this PR, you'll want to either:

1. remove the _link_ to the File Input test page
2. fix the File Input test styles